bears/python/VultureBear.py

```diff
@@ -1,25 +1,45 @@
-import re
 from shutil import which
-
+from vulture import Item, Vulture
 from coalib.bears.GlobalBear import GlobalBear
 from dependency_management.requirements.PipRequirement import PipRequirement
-from coalib.misc.Shell import run_shell_command
 from coalib.results.Result import Result
 
 
+class Unused(Item):
+
+    def __new__(cls, item, message):
+        item.message = message
+        return item
+
+
+def retrieve_unused(filenames):
+
+    def file_lineno(item):
+        return (item.filename.lower(), item.lineno)
+    unused = []
+
+    for filename in filenames:
+        vulture = Vulture()
+        vulture.scavenge([filename])
+        for item in sorted(
+                vulture.unused_funcs + vulture.unused_imports +
+                vulture.unused_props + vulture.unused_vars +
+                vulture.unused_attrs, key=file_lineno):
+            message = 'Unused %s: %s' % (item.typ, item)
+            unused.append(Unused(item, message))
+
+    return unused
+
+
 class VultureBear(GlobalBear):
     LANGUAGES = {'Python', 'Python 3'}
-    REQUIREMENTS = {PipRequirement('vulture', '0.10.0')}
+    REQUIREMENTS = {PipRequirement('vulture', '0.14.0')}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
     ASCIINEMA_URL = 'https://asciinema.org/a/82256'
     CAN_DETECT = {'Unused Code'}
 
-    EXECUTABLE = 'vulture'
-    OUTPUT_REGEX = re.compile(
-        r'(?P<filename>.*):(?P<line>.*):\s*(?P<message>.*)')
-
     @classmethod
     def check_prerequisites(cls):
```

Comment:
The check needs to be updated. Instead of `which`, we need to test importing vulture:

above:
```
try:
    from vulture import Vulture
except ImportError:
    Vulture = None
```

below:

```
@classmethod
def check_prerequisites(cls):
    return ('...' if Vulture is None else True)
```

Or is listing vulture in the `REQUIREMENTS` already enough @jayvdb ?